scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
 
     # base64 encode
-    content=$(base64 -w 0 "$path")
+    content=$(cat "$path" | base64 | tr -d '\n')
```

Comment:
The new command using `cat` and `tr` is less efficient than the original `base64 -w 0`. The original command was more direct and avoided unnecessary pipes. Consider reverting to the original implementation for better performance.